docs/stake-pool-course/handbook/grafana-dashboard-tutorial.md

```diff
@@ -450,7 +444,55 @@ $ sudo systemctl restart grafana-server
 ```
 
 
+## 8. Advanced Users: Slot Leader Panel
+![Leader Panel](/img/stake-pool-course/snsky_leaderPanel.jpg)
 
-:::note
-A panel on Leader Slots has been included, which can Alert in case the pool is selected as a leader for the next Epoch. It is a bit more complicated, so will leave it out of the tutorial, but in principle there is script running on the Producer Node which updates the leader query result in a prom file which is parsed by the node exporter, exposing this metrics to the Relay Node. For details, just drop a message on Telegram.
-:::
+Once your Pool gets big and is regularly minting blocks, it becomes diffcult to keep track of all Leader Slots and also to identify the available gaps for Pool maintainance. This Slot Leader Panel is quite helpful as it gives a good overview of all scheduled Slots in TimeSeries.
+
+
+
+Use cardano-cli to query the leadership schedule. Since the result has to interpreted by Grafana, we need to format the query output to a CSV readable syntax.
```

Comment:
actually this needs quite a visible caution warning, as most SPOs do not cater for additional RAM, the CLI query can use up more than 10GB alone, risking OOM crash for node if not sized correctly on mainnet

Reply:
that's correct. is there any documentation of min requirement that i can reference to?

Reply:
Currently, there isnt a documentation, but QA has noticed and [raised it too](https://github.com/input-output-hk/cardano-node/issues/3673), besides a number of other issues raised across IO repositories about splitting for ledger-state queries

Reply:
i can mention that i needed 16 + 8 GB to execute the query and it needed several minutes. and can link the above issue. ok?

Reply:
Sounds fair to me